lib/assets/javascripts/views/questions/edit.js

```diff
@@ -11,5 +11,10 @@ $(() => {
     questionEdit.hide();
     questionEdit.parent().find('.question_show').show();
   });
+  $('.new_question_cancel').on('click', (e) => {
```

Comment:
good catch with this. Please, rename the file accordingly since now handles a cancel button for edit form and and a cancel button for new form

Reply:
renamed